### PR TITLE
Testing: entity's creating function 

### PR DESCRIPTION
**Pull request by duylk-1584**
## Related Tickets
- [222729](https://dev.framgia.com/issues/222729)
- [222735](https://dev.framgia.com/issues/222735)

## What's this PR do ?
- [x] Create UnitTest for entity controllers

## Library
*(List package, library third party add new include version)*

- Package "nock": "^10.0.6"

## Checklist
- [x] It was tested in local success?
- [x] Fill link PR into ticket and the opposite
- [x] Note reason, scope of influence, solution into ticket
- [x] Validate UI/Model/API

### CODE REVIEW

File: test/entity.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+const expect = require('chai').expect;
+var nock = require('nock');
+const uiza = require('../lib/uiza.js')('https://apiwrapper.uiza.co');
```
Comment (NgoVanTien) [inline test/entity.js]: Xóa cái namespace đi em. Thany vào cái trong example á

Comment (NgoVanTien) [reply test/entity.js]: Thay hết mấy chổ khác lun

**Comment by NgoVanTien** [comment]
ok

File: test/controllers/entity/create-spec.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+  'name': 'Test Video',
+  'url': 'https://example.com/video.mp4',
+  'inputType': 'http',
+  'description': 'test',
+}
+
+describe('Entity-Controller', function () {
+  it('entity: create successfully', async () => {
+    nock(workspaceApiDomain)
+      .post('/api/public/v3/media/entity')
```
Comment (NgoVanTien) [inline test/controllers/entity/create-spec.js]: /api/public/v3/media/entity đưa cái này vào constant luôn em

**Review by tuanquanghpvn — APPROVED** [review]

**Pull request merged**